cmd/libsnap-confine-private/string-utils.c

```diff
@@ -267,3 +267,44 @@ void sc_string_quote(char *buf, size_t buf_size, const char *str)
 	}
 	sc_string_append_char(buf, buf_size, '"');
 }
+
+void sc_string_split(const char *string, char delimiter,
+		     char *prefix, size_t prefix_size,
+		     char *suffix, size_t suffix_size)
+{
+	if (string == NULL) {
```

Comment:
Maybe those are too evident, but we may still want to add a couple of simple test cases for (string == NULL) and (prefix == NULL && suffix == NULL)

Reply:
Thanks! Done in fc05b1b71c9c21fba9c1b39a4ddf9bc71b380e9a.